cryptotracker/templates/index.html

```diff
@@ -24,23 +24,127 @@
   <body style="background-color: black;">
 
 
-    <div class="container">
+    <div class="container" id="home">
       <div class="row">
         <ul class="menu">
-          <li class="menu-item"><a class="active" href="index.html">Bitcoin</a></li>
-          <li class="menu-item"><a href="https://www.youtube.com/watch?v=o0u4M6vppCI">Dogecoin</a></li>
-          <li class="menu-item"><a href="https://www.youtube.com/watch?v=Gx9OlBM8Sso">Ethereum</a></li>
+          <li class="menu-item"><a class="active" href="#home" onclick="displayBit()">Bitcoin</a></li>
+          <li class="menu-item"><a class="active" href="#home" onclick="displayDoge()">Dogecoin</a></li>
+          <li class="menu-item"><a class="active" href="#home" onclick="displayEther()">Ethereum</a></li>
         </ul>
       </div>
-      <div class="row">
-        <div class="col-sm-8">
-          <canvas id="chart" width=100% height=60%></canvas>
-          <script src="{{url_for('static', filename='CCGraph.js')}}"></script>
+
+      <div class="cryptocharts">
+        <script>
+          function displayBit(){
+            var B = document.getElementById("bit");
+            var D = document.getElementById("doge");
+            var E = document.getElementById("ether");
+
+           if (B.style.display == "none"){
+              B.style.display = "block";
+              D.style.display = "none";
+              E.style.display = "none";
+              document.querySelector(".bitcnv").id="chart";
+              document.querySelector(".dogecnv").id="temp";
+              document.querySelector(".ethercnv").id="temp";
+              document.getElementById("chart").remove();
+              let canvas = document.createElement('canvas');
+              canvas.setAttribute('id','chart');
```

Comment:
The chart has an update method built in - or the data can be swapped directly to the update it. That will keep us from having to destroy and rebuild the chart element every time we switch tabs:
https://www.chartjs.org/docs/latest/developers/updates.html

Related to the flashing issue so I imagine it would solve that